src/Shared/Recruit.Vacancies.Client/Application/Services/IGetVacancyTitlesProvider.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Esfa.Recruit.Vacancies.Client.Application.Services
```

Comment:
Should this be under the Providers folder?

Reply:
will move